<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorOptions</class>
 <widget class="QWidget" name="ColorOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>47</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>94</width>
    <height>47</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scene Color</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="currentBackgroundLabel_">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="setBackgroundColorPushButton_">
     <property name="text">
      <string>Scene Color</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
